apps/storefront/src/pages/Invoice/index.tsx

```diff
@@ -81,15 +79,40 @@ function Invoice() {
     ({ company }) => company.companyHierarchyInfo,
   );
   const salesRepCompanyId = useAppSelector(({ b2bFeatures }) => b2bFeatures.masqueradeCompany.id);
-  const currentCompanyId =
-    role === CustomerRole.SUPER_ADMIN && isAgenting ? +salesRepCompanyId : +companyInfoId;
 
   const { invoicePayPermission, purchasabilityPermission } = useAppSelector(rolePermissionSelector);
+  const currentCompanyId =
+    role === CustomerRole.SUPER_ADMIN && isAgenting ? +salesRepCompanyId : +companyInfoId;
```

Comment:
`Number(role === CustomerRole.SUPER_ADMIN && isAgenting ? salesRepCompanyId : companyInfoId);` 🍹

Reply:
gonna add an `eslint` rule for this